moveit_core/distance_field/include/moveit/distance_field/voxel_grid.h

```diff
@@ -61,6 +62,8 @@ template <typename T>
 class VoxelGrid
 {
 public:
+  MOVEIT_DECLARE_PTR_MEMBER(VoxelGrid);
+
```

Comment:
Since `VoxelGrid<T>` is a class template, the only way to sensibly make `shared_ptr` typedefs available is as by declaring them as member types. I also had to add a `MOVEIT_DECLARE_PTR_MEMBER` macro for that.